addons/mail/static/src/models/composer/composer.js

```diff
@@ -136,10 +141,11 @@ function factory(dependencies) {
             body = this._generateMentionsLinks(body);
             body = parseAndTransform(body, addLink);
             body = this._generateEmojisOnHtml(body);
+
             let postData = {
                 attachment_ids: this.attachments.map(attachment => attachment.id),
                 body,
-                partner_ids: this.mentionedPartners.map(partner => partner.id),
+                partner_ids: this.getMessagePartners(),
                 message_type: 'comment',
```

Comment:
alphabetical order (was not the case before PR, but you could fix that at the same time)